<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlatformSettingDialog</class>
 <widget class="QDialog" name="PlatformSettingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>386</width>
        <height>379</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>History size:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="historySize">
         <property name="minimum">
          <number>10</number>
         </property>
         <property name="maximum">
          <number>1000000</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Data rotation:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="dataRotation">
         <property name="decimals">
          <number>3</number>
         </property>
         <property name="maximum">
          <double>360.000000000000000</double>
         </property>
         <property name="value">
          <double>90.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Grid</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QFrame" name="gridFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="midLineWidth">
          <number>0</number>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <item row="0" column="2">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Y</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="minX">
            <property name="minimum">
             <number>-60000</number>
            </property>
            <property name="maximum">
             <number>60000</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QSpinBox" name="minY">
            <property name="minimum">
             <number>-60000</number>
            </property>
            <property name="maximum">
             <number>60000</number>
            </property>
            <property name="value">
             <number>-100</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Max</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Min</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>X</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="maxX">
            <property name="minimum">
             <number>-60000</number>
            </property>
            <property name="maximum">
             <number>60000</number>
            </property>
            <property name="value">
             <number>2000</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QSpinBox" name="maxY">
            <property name="minimum">
             <number>-60000</number>
            </property>
            <property name="maximum">
             <number>60000</number>
            </property>
            <property name="value">
             <number>800</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Precision</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="precisionX">
            <property name="minimum">
             <number>-60000</number>
            </property>
            <property name="maximum">
             <number>60000</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QSpinBox" name="precisionY">
            <property name="minimum">
             <number>-60000</number>
            </property>
            <property name="maximum">
             <number>60000</number>
            </property>
            <property name="value">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Max Distance</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QSpinBox" name="maxDist">
         <property name="minimum">
          <number>21</number>
         </property>
         <property name="maximum">
          <number>60000</number>
         </property>
         <property name="value">
          <number>600</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="FileSelectorWidget" name="configFile" native="true"/>
         </item>
        </layout>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Config File</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Save File</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="FileSelectorWidget" name="saveFile" native="true"/>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Area Number</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="areaNumber">
         <property name="maximum">
          <number>32</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FileSelectorWidget</class>
   <extends>QWidget</extends>
   <header>FileSelectorWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
